.github/workflows/build.yml

```diff
@@ -78,6 +78,22 @@ jobs:
         ${{env.CCACHE_SETTINGS}}
         make release-static-win64 -j4
 
+  build-arch:
+    name: 'Arch Linux'
+    runs-on: ubuntu-latest
+    container:
+      image: archlinux:latest
+    steps:
+      - name: install dependencies
+        run: pacman -Syyu --noconfirm base-devel git cmake boost openssl zeromq unbound libsodium readline expat gtest python3 doxygen graphviz hidapi libusb protobuf
+      - name: configure git
+        run: git config --global --add safe.directory '*'
```

Comment:
Why is this step needed? Isn't the checkout phase from URL, and not local directory?

Reply:
See: https://github.com/tobtoht/monero/actions/runs/12848743186/job/35826479151#step:5:71